client/main.lua

```diff
@@ -78,16 +80,30 @@ Citizen.CreateThread(function ()
                 local pos = GetEntityCoords(PlayerPedId(), true)
                 if carryPackage == nil then
                     if #(pos - vector3(packagePos.x, packagePos.y, packagePos.z)) < 2.3 then
-                        DrawText3D(packagePos.x,packagePos.y,packagePos.z+ 1, "~g~E~w~ - Pack Package")
-                        if IsControlJustReleased(0, 38) then
+			if not active then
+				DrawText3D(packagePos.x,packagePos.y,packagePos.z+ 1, "~g~E~w~ - Pack Package")
+			end
+                        if IsControlJustReleased(0, 38) and not active then

```

Comment:
Should be:
```lua
if not active then
  if IsControlJustReleased(0, 38) then
```

You will make code only look if he press `E` when is not active.